packages/deployment/app/cloudformation.template.yml

```diff
@@ -690,7 +747,21 @@ Resources:
         - Fn::GetAtt:
           - SecurityGroup
           - GroupId
+       {{# if ecs.efs.mount}}
+        - Fn::ImportValue: 
+            "{{prefix}}-EFSSecurityGroup"
+        {{/if}}
+
+      {{# if ecs.amiid}}
       ImageId: {{ecs.amiid}}
+      {{else}}
+      ImageId: 
```

Comment:
this should also be updated based on my earlier comment the mapping section

Reply:
Then we remove that proposed change and leave user deal with IAAS issues.